.github/workflows/claude-nl-suite.yml

```
@@ -1,134 +1,507 @@
-name: Claude NL suite + (optional) Unity compile
+name: Claude NL/T Full Suite (Unity live)
 
 on:
-  workflow_dispatch: {}
-
+    workflow_dispatch: {}
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)